audb/core/publish.py

```diff
@@ -70,7 +70,7 @@ def _find_media(
     db_media = db.files
     for file in set(deps.media) - set(db_media):
         media.add(deps.archive(file))
-        deps.data.pop(file)
+        deps.drop(file)
```

Comment:
For `deps.drop()` we have the same question: do we want a user to be able to delete an entry?

Reply:
I'm fine with keeping it private.

Reply:
I made it private as `audb.Dependencies._drop()`